src/overlays/actors/ovl_Demo_6K/z_demo_6k.c

```diff
@@ -812,7 +812,7 @@ void func_809691BC(Demo6K* this, PlayState* play, s32 params) {
 
     temp = Environment_LerpWeight(csAction->endFrame, csAction->startFrame, play->csCtx.frames);
 
-    this->actor.world.pos.x = (((endPos.x - startPos.x) * temp) + startPos.x);
-    this->actor.world.pos.y = (((endPos.y - startPos.y) * temp) + startPos.y);
-    this->actor.world.pos.z = (((endPos.z - startPos.z) * temp) + startPos.z);
+    this->actor.world.pos.x = LERP(startPos.x, endPos.x, temp);
+    this->actor.world.pos.y = LERP(startPos.y, endPos.y, temp);
+    this->actor.world.pos.z = LERP(startPos.z, endPos.z, temp);
```

Comment:
Looks like `VEC3F_LERPIMPDST(&this->actor.world.pos, &startPos, &endPos, temp);` also works here. Not sure if that would be preferred.

Reply:
Do you like the vec macros yourself?

I like the `SQXZ` family, maybe `VEC_SET`, but more involved code in macros idk

I've been cleaning up some math macros since they were freely introduced during decomp, but at 100% it is revealed some of them are nearly useless, or only local to some files
The z64math.h macros are eligible for cleanup and there has been mixed opinions expressed about vec macros in the past

Reply:
TBH, I don't really have a preference either way (though for the `SQ` and `DOT` ones, I'd probably prefer them taking pointers, but iirc that doesn't work all the time). I do like the small bit of documentation the macros provide, but they are also all simple enough that it isn't really needed.

Reply:
Well I'm adding this macro to my list of things to consider (along with `TRANS_TYPE_FILL_WHITE2` :sunglasses:)

Can we "overlook" this for this PR and go ahead with merging?

Reply:
Yeah that is fine :)

Reply:
In the future I'll just assume an approval with comments means we can merge without waiting for the comments, to make communication less clunky (so if you have comments that should be addressed don't approve right away)